client/components/ship_pay.jsx

```diff
@@ -1,27 +1,134 @@
 import React from 'react'
-import ShipHand from './ship_hand.jsx';
+import styled from 'styled-components'
 import ReturnPolicy from './return_policy.jsx'
 
 const ShipPay = (props) => {
   console.log('inside shippay', props)
   return (
-    <div>
+    <ShipContainer>
       <p>Seller assumes all responsibility for this listing.</p>
-      <ShipHand details={props.payment}/>
+      <Shipping>
+        <Header>Shipping and Handling</Header>
+        <p>Item location <strong>{props.payment.ship_handling.item_location}</strong></p>
```

Comment:
If you use the airbnb ESLint style guide it enforces object destructuring which can make for cleaner code. It helps a lot with props in React. By destructuring you can reference this variable directly as "item_location" instead of having the long chain of object keys.